_pytest/cacheprovider.py

```diff
@@ -14,7 +14,8 @@
 class Cache(object):
     def __init__(self, config):
         self.config = config
-        self._cachedir = config.rootdir.join(".cache")
+        cache_dirname = config.getini("cache_dirname")
+        self._cachedir = config.rootdir.join(cache_dirname)
```

Comment:
Hmm I'm not sure, unless I'm missing something (and please clarify me if I am), wouldn't be just as easy to support both relative and absolute paths as I mentioned above?

I like the idea of supporting relative and absolute paths because we can eventually change the default to `$XDG_CACHE_DIR` as discussed in #1089.